### DISCUSSION TITLE
My-CS-Notebook/ML/%E7%BB%9F%E8%AE%A1%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A001%EF%BC%9A%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%E4%B8%AD%E7%9A%84%E5%9F%BA%E6%9C%AC%E6%A6%82%E5%BF%B5/

### DISCUSSION THREAD

**Discussion by giscus** [Zhang-Each/My-CS-Notebook#8] [General]
# My-CS-Notebook/ML/%E7%BB%9F%E8%AE%A1%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A001%EF%BC%9A%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%E4%B8%AD%E7%9A%84%E5%9F%BA%E6%9C%AC%E6%A6%82%E5%BF%B5/

https://zhang-each.github.io/My-CS-Notebook/ML/%E7%BB%9F%E8%AE%A1%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A001%EF%BC%9A%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%E4%B8%AD%E7%9A%84%E5%9F%BA%E6%9C%AC%E6%A6%82%E5%BF%B5/

**Comment by PurelyRookie** [comment]
complexity 那里是不是笔误了？‘过拟合其实就是对测试数据拟合得太好而对训练数据拟合效果不好’，overfitting不应该是对训练集拟合过度而在测试集表现差嘛？

**Reply by Zhang-Each** [reply]
对的对的，感谢指出。

**Reply by Zhang-Each** [reply]
后面我会修改这部分内容
